code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -124,12 +125,12 @@
 
 /client/proc/vampire_disease()
 	set category = "Vampire"
-	set name = "Diseased Touch (100)"
+	set name = "Diseased Touch"
```

Comment:
Where can the user see the cost if not here? And why would you make it less expensive?